themes/bootstrap3/templates/record/hold.phtml

```diff
@@ -66,12 +66,18 @@
 
   <?php if (in_array("pickUpLocation", $this->extraHoldFields)): ?>
     <?php
+      // Empty string as home library means that default pick up location should be
+      // used. null means that there should be no default
```

Comment:
While we're refactoring this anyway, it seems like it might be an opportunity to simplify the line below to:

```
if (($this->gatheredDetails['pickUpLocation'] ?? '') !== '') {
```

...and an equivalent change can also be made to the storage retrieval template.

Reply:
Good idea, now done!